### PR TITLE
License and checkstyle

### PR DESCRIPTION
**Pull request by sumitnigamcs**
@tivv  @rmstar @ksahilCF Below are local logs after license and codestyle changes.

_[INFO] --- apache-rat-plugin:0.13:check (rat-check) @ cdap-e2e-framework ---
[INFO] Added 1 additional default licenses.
[INFO] Enabled default license matchers.
[INFO] Added 1 custom approved licenses.
[……
[INFO] Rat check: Summary over all files. Unapproved: 0, unknown: 0, generated: 0, approved: 29 licenses.

[INFO] --- maven-checkstyle-plugin:2.17:check (validate) @ cdap-e2e-framework ---
[INFO] Starting audit...
Audit done._

### CODE REVIEW

File: src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
 package io.cdap.e2e.pages.actions;
 
 import io.cdap.e2e.pages.locators.CdfStudioLocators;
-import org.openqa.selenium.support.PageFactory;
 import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.support.PageFactory;
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java:22]: fix indent in this file

File: src/main/java/io/cdap/e2e/pages/actions/CdfGcsActions.java [hunk 2]
```diff
@@ -1,49 +1,76 @@
...
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.PageFactory;
 
+import java.io.IOException;
+import java.util.UUID;
+
+/**
+ * Represents CdfGcsActions
+ */
 public class CdfGcsActions {
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/pages/actions/CdfGcsActions.java:32]: fix indent in this file

**Review by rmstar — COMMENTED** [review]
please fix use cask style in all files

File: src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java [hunk 3]
```diff
@@ -1,31 +1,50 @@
...
+  static {
+
+    cdfBigQueryPropertiesLocators =
+      PageFactory.initElements(SeleniumDriver.getDriver(),
+                               io.cdap.e2e.pages.locators.CdfBigQueryPropertiesLocators.class);
+  }
+
+  public static void enterBigQueryProperties(String arg0) throws InterruptedException {
+    CdfStudioLocators.bigQueryProperties.click();
+    io.cdap.e2e.pages.locators.CdfBigQueryPropertiesLocators.bigQueryReferenceName.sendKeys("automation_test");
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/pages/actions/CdfBigQueryPropertiesActions.java:39]: fix indent here and in other places in this file

File: src/main/java/io/cdap/e2e/pages/actions/CdfSysAdminActions.java [hunk 4]
```diff
@@ -1,51 +1,72 @@
...
+       System.out.println("Succeess");
+     }
 
-            Assert.assertTrue(checkParam,true);
-    }
+     Assert.assertTrue(checkParam, true);
+   }
 
-}
+ }
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/pages/actions/CdfSysAdminActions.java]: why is there a space before } ?

File: src/main/java/io/cdap/e2e/pages/actions/GoogleSignInActions.java [hunk 5]
```diff
@@ -1,35 +1,55 @@
...
+  public static io.cdap.e2e.pages.locators.GoogleSignInLocator googleSignInLocator = null;
+
+  static {
+
+    googleSignInLocator = PageFactory.initElements(SeleniumDriver.getDriver(), GoogleSignInLocator.class);
+
+  }
+
+  public static void login() throws InterruptedException, IOException {
+    googleSignInLocator.username.get(0).sendKeys(SeleniumHelper.readParameters("gLoginUsername"));
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/pages/actions/GoogleSignInActions.java:37]: fix ident here and in other places in this file

File: src/main/java/io/cdap/e2e/utils/SeleniumDriver.java [hunk 6]
```diff
@@ -11,94 +26,94 @@
...
+    // System.out.println("Check thos one------"+session);
+    return session;
+  }
+
+  public static void getWaitDriver() {
+    new WebDriverWait(chromeDriver, 40000).
+      until(ExpectedConditions.invisibilityOf(chromeDriver.findElement(By.className("loading-bar"))));
+
+  }
+
```
Comment (rmstar) [inline src/main/java/io/cdap/e2e/utils/SeleniumDriver.java:75]: remove newline

File: src/main/java/stepsdesign/AfterActions.java [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+public class AfterActions {
+
+  @AfterStep
+  public static void tearDown(Scenario scenario) {
+    WebDriver driver = SeleniumDriver.getDriver();
+    byte[] screenshotBytes = ((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
+    scenario.embed(screenshotBytes, "image/png");
+
+  }
+
```
Comment (rmstar) [inline src/main/java/stepsdesign/AfterActions.java]: remove newline

File: src/main/java/stepsdesign/BeforeActions.java [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+/**
+ * Represents BeforeActions
+ */
+public class BeforeActions {
+  public static Scenario scenario;
+  public static File myObj;
+
+  @Before
+  public void setUp(Scenario scenario) throws IOException {
+    //ExtentCucumberFormatter.initiateExtentCucumberFormatter();
```
Comment (rmstar) [inline src/main/java/stepsdesign/BeforeActions.java]: remove commented out line

File: src/main/java/stepsdesign/MyStepdefs.java [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package stepsdesign;
+
+/**
+ * Represents MyStepDefs
+ */
+public class MyStepdefs {
```
Comment (rmstar) [inline src/main/java/stepsdesign/MyStepdefs.java]: Is this class used anywhere?

File: src/main/java/io/cdap/e2e/pages/actions/GoogleSignInActions.java (hunk 5 above)
Comment (sumitnigamcs) [reply src/main/java/io/cdap/e2e/pages/actions/GoogleSignInActions.java:37]: sorry I didn't understand this one. Please specify @rmstar

Comment (rmstar) [reply src/main/java/io/cdap/e2e/pages/actions/GoogleSignInActions.java:37]: sorry, please ignore this comment. I was confused by github's diff formatting

**Pull request merged**